contentcuration/contentcuration/static/js/utils/studioJquery.js

```diff
@@ -0,0 +1,56 @@
+var $ = require('rawJquery');
```

Comment:
Almost went for a modules alias, mapping directly to node_modules, but thought it was kind of heavy handed. @rtibbles any thoughts on this?